client/grandpa-ranger/package.json

```
@@ -11,8 +10,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@polkadot/api": "=7.14.3",
-    "@t3rn/types": "=0.1.1",
     "@types/debug": "=4.1.7",
+    "axios": "^0.27.2",
```

```suggestion
    "axios": "=0.27.2",
```
developed dis auto update paranoia..would go for a pinned npm version